tests/pages/course-page.js

```diff
@@ -105,7 +105,7 @@ export default create({
       scope: '[data-test-badge-earned-modal]',
     },
 
-    clickOnViewButton: clickable('[data-test-view-button]'),
+    clickOnViewButton: clickable(),
```

Comment:
There's no "view" button now no? Can remove this and just use `earnedBadgeNotice.click()`.